src/dune_rules/run_with_path.ml

```diff
@@ -8,6 +8,7 @@ let depexts_hint = function
     ; Pp.enumerate ~f:Pp.verbatim depexts
     ]
     |> Pp.concat_map ~sep:Pp.cut ~f:(fun pp -> Pp.box pp)
```

Comment:
This should be `~f:Fun.id` and you should put `Pp.hovbox` on the first line and `Pp.vbox` on the second. Then you should do a `Pp.vbox` after concatenating. I think this will give you what you want.